san/graphql.py

```diff
@@ -9,10 +9,13 @@ def execute_gql(gql_query_str):
     if ApiConfig.api_key:
         headers = {'authorization': "Apikey {}".format(ApiConfig.api_key)}
 
-    response = requests.post(
-        SANBASE_GQL_HOST,
-        json={'query': gql_query_str},
-        headers=headers)
+    try:
+        response = requests.post(
+            SANBASE_GQL_HOST,
+            json={'query': gql_query_str},
+            headers=headers)
+    except requests.exceptions.RequestException as e:
+        raise SanError(f'Error running query: ({e})')
```

Comment:
Do we use f-strings syntax in other places. Using it here means that we start to require python 3.6 in order to use sanpy.
If not used anywhere else - lets continue to use `.format` syntax.

Reply:
Got used to use f' syntax, you're right about it, we should be consistent